### ISSUE TITLE
md2@0.0.5 and 0.0.6 dont't work on @angular 2.1.0 with meteor 1.4

### ISSUE THREAD

**Issue by omararturo** [Promact/md2#27]
When run the meteor app run well, but in the browser the error appear:

**Uncaught SyntaxError: Unexpected token import**

The error is in this line:

**import { ApplicationRef, ComponentFactoryResolver, Injectable, ReflectiveInjector, NgModule } from '@angular/core';**

this is the fragment of code when the error appear:

```
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//                                                                                                                     //
// node_modules/md2/toast/toast.js                                                                                     //
//                                                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
                                                                                                                       //
var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {                               // 1
    var c = arguments.length, r = c < 3 ? target : desc === null ? desc = Object.getOwnPropertyDescriptor(target, key) : desc, d;
    if (typeof Reflect === "object" && typeof Reflect.decorate === "function") r = Reflect.decorate(decorators, target, key, desc);
    else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
    return c > 3 && r && Object.defineProperty(target, key, r), r;                                                     // 5
};                                                                                                                     // 6
var __metadata = (this && this.__metadata) || function (k, v) {                                                        // 7
    if (typeof Reflect === "object" && typeof Reflect.metadata === "function") return Reflect.metadata(k, v);          // 8
};                                                                                                                     // 9
import { ApplicationRef, ComponentFactoryResolver, Injectable, ReflectiveInjector, NgModule } from '@angular/core';    // 10
import { CommonModule } from '@angular/common';                                                                        // 11
import { Md2ToastComponent } from './toast.component';                                                                 // 12
export var Toast = (function () {                                                                                      // 13
    function Toast(message) {
```

**Comment by dharmeshpipariya-zz** [comment]
We have fixed this issue with release md2@0.0.7,
comment here if you have an issue

**Comment by omararturo** [comment]
Hi @dharmeshpipariya,
Same issue in md2@0.0.7

**Comment by dharmeshpipariya-zz** [comment]
We have upgraded dependencies with release md2@0.0.10-3,
comment here if you have an issue

**Comment by dharmeshpipariya-zz** [comment]
We have fixed the issue with release md2@0.0.12,
comment here if you have an issue

**Comment by filosof86** [comment]
Hi @dharmeshpipariya, 

Sorry for reopening, but I keep getting the same error (Uncaught SyntaxError: Unexpected token import) while importing the Md2Module into my application.

The browser debug console states the error is the following string

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//                                                                                                                     //
// node_modules/md2/index.js                                                                                           //
//                                                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
                                                                                                                       //
---->>>> export * from './core';                                                                                                // 1
export * from './module';                                                                                              // 2
export * from './accordion/index';                                                                                     // 3
export * from './autocomplete/index';                                                                                  // 4
......


MD2 version: "md2@0.0.28"

Meteor version: meteor@1.5.1

Could you please help me here?

Thank you in advance
